custom_dc/data_cloud.md

```diff
@@ -192,13 +205,26 @@ docker run \
 -v <var>OUTPUT_DIRECTORY</var>:<var>OUTPUT_DIRECTORY</var> \
 -e GOOGLE_APPLICATION_CREDENTIALS=/gcp/creds.json \
 -v $HOME/.config/gcloud/application_default_credentials.json:/gcp/creds.json:ro \
+[-e DATA_RUN_MODE=schemaupdate \]
```

Comment:
What's the reason for including this here but not in the matching command snippet on the local_data page? My preference is to remove it in both places because it isn't relevant in most cases.